rust/operator-binary/src/airflow_controller.rs

```diff
@@ -965,9 +977,23 @@ fn build_server_rolegroup_statefulset(
         };
         airflow_container.readiness_probe(probe.clone());
         airflow_container.liveness_probe(probe);
-        airflow_container.add_container_port("http", resolved_port.into());
+        airflow_container.add_container_port(HTTP_PORT_NAME, http_port.into());
     }
 
+    let listener_class = &merged_airflow_config.listener_class;
+    // all listeners will use ephemeral volumes as they can/should
+    // be removed when the pods is re-started, and no data needs to be preserved
```

Comment:
```suggestion
    // be removed when the pods is re-started, and no data needs to be preserved
```
Ephemeral PVCs survive pod restarts (they are not recreated)

Reply:
5c7022c